plugins/rapid7_insightidr/komand_rapid7_insightidr/util/formatting.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+
+def refactor_message(message: dict):
```

Comment:
Please add the `Dict` type from `typing` i.e. `Dict[str, str]` and the return typehint. Kindly please add the doc-string as well 😄